pull/github_context.go

```diff
@@ -170,7 +170,7 @@ func (ghc *GithubContext) RequiredStatuses(ctx context.Context) ([]string, error
 		}
 	}
 	if checks := ghc.branchProtection.GetRequiredStatusChecks(); checks != nil {
-		return checks.Contexts, nil
+		return *checks.Contexts, nil
```

Comment:
I think we should use `GetContexts()` here instead of assuming that `checks.Contexts` is non-nil